CHANGELOG.md

```
@@ -7,14 +7,1442 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+  - `subindexmap_get_refines` - SubIndexMap::get matches sim_get
+  - `subindexmap_insert_refines` - SubIndexMap::insert matches sim_set
+  - `hashmap_get_steps` - HashMap get stepping
+  - `subindexmap_get_steps` - SubIndexMap get stepping
+  - `stemnode_new_is_empty` - StemNode::new produces empty SubIndexMap
+  - `or_insert_with_steps` - or_insert_with closure evaluation
+  - `hashmap_entry_steps` - HashMap entry stepping
+  - `subindexmap_insert_steps` - SubIndexMap insert stepping
+  - `tree_rebuild_preserves_refines` - refinement preservation via reflexivity
+  - `slice_iter_steps` (iterator_stepping.v) - slice iteration stepping
+
+### Added
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)